src/test/java/com/example/moviemanager/repository/MovieRepositoryTest.java

```diff
@@ -0,0 +1,83 @@
+package com.example.moviemanager.repository;
+
+import com.example.moviemanager.repository.model.Movie;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DataJpaTest
+@ActiveProfiles("test")
+public class MovieRepositoryTest {
+
+    @Autowired
+    private MovieRepository movieRepository;
+
+    @Test
+    void return_collection_of_movies() {
+        // given
+        movieRepository.save(new Movie("Home Alone", "Christmas movie", 1990, 8.5));
+        movieRepository.save(new Movie("Home Alone 2", "Christmas movie 2", 1992, 8.8));
+
+        // when
+        List<Movie> actualMovies = movieRepository.findAll();
+        List<Long> moviesId = List.of(actualMovies.get(0).getId(), actualMovies.get(1).getId());
+
+        // then
+        assertTrue(moviesId.contains(1L) && moviesId.contains(2L));
```

Comment:
These are two assertions put under one. Please make separate assertions. What would be event better is to use assertJ library which has nice assertion methods which can be chained. For example such assertion could be rewritten with AssertJ like that:
```
assertThat(actualMovies)
                .extracting("id")
                .containsExactly(1L, 2L);
```